collector/loadavg_unix.go

```diff
@@ -11,6 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build darwin dragonfly freebsd netbsd openbsd solaris
 // +build !noloadavg
 // +build !linux
```

Comment:
Since you added a line that specifies all OSes to build for explicitly, you can remove this `+build !linux` constraint.